### PR TITLE
fix(style) : Setting section success messages border color

### PR DESCRIPTION
**Pull request by anweshknayak**
fixes https://github.com/mozilla/fxa/issues/272#issuecomment-363217030

@vladikoff

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 43e9ea4ac272ffd9ce0fa1efd9ecda3a4f0ec7f1
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a179c8f240bfc8b230a881befad8fd7f4bdecaea
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: app/styles/_variables.scss [hunk 1]
```diff
@@ -98,7 +98,7 @@ $modal-background-color-iframe: $color-white;
 //Settings Specific Colors
 $settings-header-border-bottom: $grey-40;
 $settings-success-background-color: $color-green-darker;
-$settings-success-border-color: $color-green-dark;
+$settings-success-border-color: #058b00;
```
Comment (vladikoff) [inline app/styles/_variables.scss]: I think u need to use `$settings-success-background-color` here

**Comment by vladikoff** [comment]
@anweshknayak also in the commit message you can do `fix(style): ...` , it seems empty at the moment

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a179c8f240bfc8b230a881befad8fd7f4bdecaea
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 9273606eae5a482737b44700ce8a5c16ee792134
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by anweshknayak** [comment]
@vladikoff done

**Comment by vladikoff** [comment]
Thank you @anweshknayak !

**Comment by anweshknayak** [comment]
thanks a lot @vladikoff for your support. Really appreciate it 🎉

**Pull request merged**